File: Programmers/Lv.0/숫자 찾기/120904.java
class Solution {
    public int solution(int num, int k) {
        int answer = -1;
        
        String[] str = String.valueOf(num).split("");
        for (int i=0; i<str.length; i++) {
            if (str[i].equals(String.valueOf(k))) {
                answer = i + 1;
                break;
            }
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/숫자 찾기/120904.js
function solution(num, k) {
    let answer = -1;
    
    const str = [...num.toString()];
    for (let i=0; i<str.length; i++) {
        if (str[i] === k.toString()) {
            answer = i + 1;
            break;
        }
    }
    
    return answer;
}
